backend/database.py

```
@@ -811,56 +892,159 @@ def cleanup_test_data() -> Dict[str, int]:
 
     Returns:
         dict[int, dict]: Mapping of faiss_idx to file metadata dictionary.
-
-    Raises:
-        ValueError: If too many indices are provided for a single SQLite query.
     """
     if not indices:
         return {}
 
-    # Deduplicate and validate
     unique_indices = sorted(list(set(indices)))
```

Ruff (0.15.20)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)